skate-plugin/project-gen/src/jvmMain/kotlin/slack/tooling/aibot/Message.kt

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (C) 2024 Slack Technologies, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package slack.tooling.aibot
+
+import androidx.compose.runtime.Immutable
+
+// not sure if I need this in a separate class
```

Comment:
Could you expand on this?

Reply:
Do you think it's okay to have the Message data class in a separate file? Or is it unnecessary?

Reply:
if you're going to reuse it yeah, otherwise you could just co-locate it with the rest of your state models